website/docs/r/pki_secret_backend_role.html.md

```diff
@@ -112,7 +112,14 @@ The following arguments are supported:
 
 * `require_cn` - (Optional) Flag to force CN usage
 
-* `policy_identifiers` - (Optional) Specify the list of allowed policies IODs
+* `policy_identifiers` - (Optional) Specify the list of allowed policies OIDs; Deprecated: use `policy_identifier` blocks instead
```

Comment:
I don't believe we have deprecated this field yet. We probably just want to say that it is used for Vault 1.10 and below, or something like that.